base/config_persistence.go

```diff
@@ -108,7 +108,7 @@ func (xbp *XattrBootstrapPersistence) loadRawConfig(ctx context.Context, c *gocb
 		xattrContErr := res.ContentAt(0, &rawValue)
 		if xattrContErr != nil {
 			DebugfCtx(ctx, KeyCRUD, "No xattr config found for key=%s, path=%s: %v", key, cfgXattrConfigPath, xattrContErr)
-			return rawValue, 0, ErrNotFound
```

Comment:
I am very reluctant to change this error type because this has an impact on the retry behavior - the usage of `IsDocNotFoundError` in each call in `config_manager.go` depends on this https://github.com/couchbase/sync_gateway/blob/aaa9f0136925df4469b36f4919fa932b381089be/rest/config_manager.go#L438

I'd feel safer about _wrapping_ this `ErrNotFound` vs retyping this error but I do not know if this error would bubble up in this case.